<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeviceKinectDK</class>
 <widget class="QDockWidget" name="DeviceKinectDK">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>260</width>
    <height>261</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>260</width>
    <height>261</height>
   </size>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 10pt &quot;微软雅黑&quot;;</string>
  </property>
  <property name="windowTitle">
   <string>Kinect DK</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 9pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>3</number>
       </property>
       <property name="leftMargin">
        <number>6</number>
       </property>
       <property name="topMargin">
        <number>6</number>
       </property>
       <property name="rightMargin">
        <number>6</number>
       </property>
       <property name="bottomMargin">
        <number>6</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,3">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Depth Mode:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cbox_depth">
           <item>
            <property name="text">
             <string>NFOV_2X2BINNED </string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>NFOV_UNBINNED </string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>WFOV_2X2BINNED </string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>WFOV_UNBINNED </string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Image Format:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cbox_colorformat">
           <item>
            <property name="text">
             <string>MJPG </string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>NV12 </string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>YUY2 </string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>BGRA32 </string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Color Resolution:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cbox_resolution">
           <item>
            <property name="text">
             <string>720P </string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>1080P</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>1440P</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>1536P </string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2160P</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>3072P </string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="0,0">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Output Type:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cbox_output">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <item>
            <property name="text">
             <string>PointCloud</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>ColorImage</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>DepthImage</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>InfraredImage</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Camera FPS:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rbtn_5fps">
           <property name="text">
            <string>5</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rbtn_15fps">
           <property name="text">
            <string>15</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rbtn_30fps">
           <property name="text">
            <string>30</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="1,1,2,1">
         <item>
          <widget class="QCheckBox" name="check_withoutcolor">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../src.qrc">
             <normaloff>:/icon/resource/icon/channelmixer.svg</normaloff>:/icon/resource/icon/channelmixer.svg</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="check_autofresh">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../src.qrc">
             <normaloff>:/icon/resource/icon/real-time.svg</normaloff>:/icon/resource/icon/real-time.svg</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="slider_refreshtime">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimum">
            <number>30</number>
           </property>
           <property name="maximum">
            <number>3000</number>
           </property>
           <property name="singleStep">
            <number>100</number>
           </property>
           <property name="value">
            <number>30</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="txt_refreshtime">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>0.030 s</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="styleSheet">
          <string notr="true">border:2px solid lightgray</string>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="2,3,3">
         <property name="spacing">
          <number>6</number>
         </property>
         <item>
          <widget class="QPushButton" name="btn_add">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Add</string>
           </property>
           <property name="icon">
            <iconset resource="../src.qrc">
             <normaloff>:/icon/resource/icon/add.svg</normaloff>:/icon/resource/icon/add.svg</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_connect">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Connect</string>
           </property>
           <property name="icon">
            <iconset resource="../src.qrc">
             <normaloff>:/icon/resource/icon/connect.svg</normaloff>:/icon/resource/icon/connect.svg</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_capture">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Capture</string>
           </property>
           <property name="icon">
            <iconset resource="../src.qrc">
             <normaloff>:/icon/resource/icon/start.svg</normaloff>:/icon/resource/icon/start.svg</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../src.qrc"/>
 </resources>
 <connections/>
</ui>
